kpatch-build/kpatch-build

```diff
@@ -649,6 +658,26 @@ module_name_string() {
 	echo "${1//[^a-zA-Z0-9_-]/-}" | cut -c 1-55
 }
 
+is_supported_distro(){
```

Comment:
Ubuntu and debian are also supported, which isn't reflected here.  Better to call it `is_supported_rpm_distro()` or so.

Reply:
I move ubuntu and debian into SUPPORTED_RPM_DISTROS. The vmlinux location is different of these tow release. Therefore, I made some changes from line 901 to line 918. :-)